rmw_cyclonedds_cpp/CMakeLists.txt

```diff
@@ -32,7 +32,12 @@ find_package(ament_cmake_ros REQUIRED)
 find_package(rcutils REQUIRED)
 
 find_package(cyclonedds_cmake_module REQUIRED)
-find_package(CycloneDDS REQUIRED CONFIG)
+find_package(CycloneDDS QUIET CONFIG)
+if(NOT CycloneDDS_FOUND)
+  message(WARNING "Could not find ADLINK CycloneDDS - skipping '${PROJECT_NAME}'")
```

Comment:
It isn't "ADLINK CycloneDDS": it is "Eclipse Cyclone DDS". The "Eclipse" bit really matters, but if it needs changing, then the space between "Cyclone" and "DDS" might as well be added, too.

(BTW: what is the appropriate way of dealing with such trivial changes? A comment like this, or just changing it in the editer? Just curious what in your experience is the best way to handle them.)

Reply:
I just changed it to "Eclipse Cyclone DDS".
For changes like these, feel free to push directly on the branch to reflect your changes.